api/serializers/student.py

```diff
@@ -56,4 +68,27 @@ class Meta:
             "status",
             "is_member_of_speaking_club",
             "teaching_languages_and_levels",
+            "non_teaching_help_required",
         )
+
+
+class PublicStudentSerializer(CommonPublicStudentSerializer):
+    """Representation of a Student that is used in 'All students' Tooljet view."""
+
+    class Meta(CommonPublicStudentSerializer.Meta):
+        pass
+
+
+class PublicStudentWithPersonalInfoSerializer(CommonPublicStudentSerializer):
+    """
+    Representation of a Student with personal info that is used
+    in 'Students by coordinator' Tooljet view.
+    """
+
+    personal_info = PublicPersonalInfoSerializer(read_only=True)
+    non_teaching_help_required = NonTeachingHelpSerializerField()
```

Comment:
I guess this should be removed, since it's in Common now?

Reply:
Fixed